Add more unit test for DynamicNFT (#2892)
XRPLF/xrpl.js: packages/xrpl/HISTORY.md

* New `MPTAmount` type support for `Payment` and `Clawback` transactions
* `parseTransactionFlags` as a utility function in the xrpl package to streamline transactions flags-to-map conversion
* Support for XLS-70d (Credentials)

> * Add `NFTokenModify` transaction and add `tfMutable` flag in `NFTokenMint` (XLS-46)

Reviewer: ```suggestion
* Add support for the `DynamicNFT` amendment (XLS-46)
```

Author: updated
